### ISSUE TITLE
Own Model Training regarding

### ISSUE THREAD

**Issue by niyaz1996** [TJCVRS/CRNN_Tensorflow#35]
I downloaded the file under the folder named CRNN_Tensorflow-master
And I kept my dataset under the name CRNN_Tensorflow-master\imagesdataset
And I kept the sample.txt file in CRNN_Tensorflow-master but when I try to run the write_features.py code it  shows an assetion error as " assert ops.exists(self.__train_dataset_dir)"

Please let me know and Thanks in Advance!

**Comment by TJCVRS** [comment]
@niyaz1996 You might orgnize your own dataset in a wrong way, please follow the readme file to reorgnize your own dataset

**Comment by niyaz1996** [comment]
Yes!It's working for me!Thank you very much!

**Comment by niyaz1996** [comment]
And I want to know whether t
![retail1](https://user-images.githubusercontent.com/36569431/36371662-cdf3b42a-1588-11e8-9c1c-5d065e192f69.jpg)
his net can be applied to retail images to extract the text from it! I attached an example with this message and I want to extract some text(probably BIOELEMENTS/ any other text) that I see .

Thanks in advance!

**Comment by zxy3416391** [comment]
@niyaz1996 Hello,I wonder how to orgnize  own dataset,should we divide the dataset into train,test,validation ourselves?can you show me the form of the dataset file

**Comment by odgiv** [comment]
I have the same problem. In data_provider.py, it asserts dataset_dir has sub dir 'Train'. But in the readme file, there is no instruction to have Train or Test subfolders.
```
self.__train_dataset_dir = ops.join(self.__dataset_dir, 'Train')
self.__test_dataset_dir = ops.join(self.__dataset_dir, 'Test')

assert ops.exists(dataset_dir)
```

**Comment by jetjodh** [comment]
@niyaz1996 @odgiv  @zxy3416391 Did you make subfolders? Did it solve your problem?

**Comment by odgiv** [comment]
@jetjodh Yes. You have to create Train and Test subfolders inside dataset dir. Put images and sample.txt in them. 
[Like here](https://github.com/MaybeShewill-CV/CRNN_Tensorflow/issues/106)
